packages/smooth_app/lib/data_models/user_preferences.dart

```diff
@@ -19,6 +19,10 @@ class UserPreferences extends ChangeNotifier {
   static const String _TAG_THEME_DARK = 'themeDark';
   static const String _TAG_THEME_COLOR_TAG = 'themeColorTag';
 
+  bool isFirstTimeUser() {
+    return _sharedPreferences.getBool(_TAG_INIT) == null;
+  }
+
```

Comment:
I'm sure you could use this method 5 lines below in the code.

Reply:
ummm. which one ?

Reply:
ah, sorry misread your original comment. Just read it again, makes sense.